packages/labs/virtualizer/src/test/scenarios/element-and-directive-parity.test.ts

```diff
@@ -168,6 +168,10 @@ describe('lit-virtualizer and virtualize directive', () => {
     expect(uvd.shadowRoot?.textContent).not.to.include('[5 selected]');
 
     // Clearing event arrays so we can watch for specific future events.
+    await until(() => ulv.rangeChangedEvents.length > 0);
```

Comment:
Adding these four lines was an opportunistic fix for a flaky test that was advancing to the event clearing step before ensuring all events were received.

Reply:
ah nice! I'd been noticing some flakiness from the virtualizer tests

Reply:
I'm going to try and address them where I find them -- mostly I encounter them during the CI automated checks because the environments are slower than my workstations... if you see any others me a msg and I'll try to fix it.  I definitely don't want Virtualizer flakes to slow down anyone's flow! :)